Modules/HTTP/Sources/HTTP/Middleware/ContentNegotiationMiddleware/ClientContentNegotiatonMiddleware.swift

```diff
@@ -0,0 +1,74 @@
+public struct ClientContentNegotiationMiddleware : Middleware {
```

Comment:
This name is ridiculously long. Any ideas on how to shorten it? Maybe make a `Middleware` namespace?